### DISCUSSION TITLE
Welcome to react-echarts Discussions!

### DISCUSSION THREAD

**Discussion by hugocxl** [hugocxl/react-echarts#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ashishsurya** [comment]
Can we use it in production, instead of [echarts-for-react](https://git.hust.cc/echarts-for-react/)

**Reply by hugocxl** [reply]
hi @ashishsurya! sure, it's already being used in several projects. Just mind: if you're using Next.js check the open issue in the repo regardless the "app" router, and how can it be used in that case.
